app/models/post.rb

```diff
@@ -0,0 +1,19 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+
+  has_many :comments, foreign_key: 'post_id', dependent: :destroy
+  has_many :likes, foreign_key: 'post_id', dependent: :destroy
+
+  after_save :update_user_posts_counter
+
+  def recent_comments
+    Comment.where(post_id: id).order(created_at: :desc).limit(5)
```

Comment:
:orange_circle: [OPTIONAL] Below is a more concise and clear way to achieve the same result as the previous code: 
```
def recent_comments
    comments.order(created_at: :desc).limit(5)
 end
```

 This leverages ActiveRecord associations, using the  has_many :comments association set up in this model.